clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -211,6 +211,7 @@ protected synchronized boolean ensureCoordinatorReady(long startTimeMs, long tim
                     if (remainingMs <= 0)
                         break;
 
+                    log.debug("Coordinator discover failed for group {}, refresh metadata and try again", groupId);
```

Comment:
Nit: did you mean `discovery`? Also, would `refreshing metadata and trying again` be more appropriate given that we are actually doing that instead of telling someone to do it?